Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -313,5 +874,24 @@ private void RenderToggle(MRConfig configKey, string title)
                 }
             }
         }
+
+        private void CreateSpace(float width)
+        {
+#if UNITY_2019_3_OR_NEWER
+            EditorGUILayout.Space(width);
+#else
+            for (int i = 0; i < Math.Ceiling(width / 5); i++)
+            {
+                EditorGUILayout.Space();
+            }
+#endif // UNITY_2019_3_OR_NEWER
+        }
+#if UNITY_2019_3_OR_NEWER
+        private void AddUPMPackage(string packageName)
+        {
+            var request = UnityEditor.PackageManager.Client.Add(packageName);
+            while (!request.IsCompleted) { }
```

Comment:
Do we want to provide some sort of spinning UI here, to let the use know something is happening?

Reply:
Great suggestion, but personally I would like to add that in a later PR - currently after clicking the button Unity Editor's UI will suggest to user that something is going on.